### ISSUE TITLE
QA_EXPT-7 & EXPT-8_Procedure_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#24]
Defect Description :
In the Procedure page of 7th and 8th experiments of this lab, found that the content is same.

Actual Result :
In the Procedure page of 7th and 8th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i21](https://cloud.githubusercontent.com/assets/13479177/26663884/6adb90bc-46ab-11e7-8d81-af10d52b037b.png)

**Comment by sravanthimodepu** [comment]
Yes.. The 7th & 8th experiments belongs to the same content but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
